index.bs

```diff
@@ -1524,17 +1532,36 @@ When this method is invoked, the [=authenticator=] must perform the following pr
     <figcaption>Generating an [=assertion signature=].</figcaption>
 </figure>
 
-On successful completion, the authenticator returns to the user agent:
+On successful completion, the [=authenticator=] returns to the user agent:
 - The identifier of the credential (credential ID) used to generate the [=assertion signature=].
 - The [=authenticator data=] used to generate the [=assertion signature=].
 - The [=assertion signature=].
 
-If the authenticator cannot find any credential corresponding to the specified [=[RP]=] that matches the specified criteria, it
+If the [=authenticator=] cannot find any credential corresponding to the specified [=[RP]=] that matches the specified criteria, it
 terminates the operation and returns an error.
 
 If the user refuses consent, the authenticator returns an appropriate error status to the client.
+    </dd>
+    <dt>"public-key-uaf"</dt>
+    <dd>
+- Follow the procedure as specified in [[FIDO-UAF-Authenticator-Commands]] section 6.3. Using
+    * The caller's RP ID as "AppID"
```

Comment:
RP ID is only a hostname in webauthn (at this time).  In UAF, the RP server supplies the AppID in the OperationHeader.  Perhaps we should use the AppID extension to allow the RP to send its AppID and then pass that down to the authnr?  Though, the AppID extension has issues, see issue #491